python/pyspark/sql/utils.py

```diff
@@ -16,6 +16,10 @@
 #
 
 import py4j
+import sys
+
+if sys.version_info.major >= 3:
+    unicode = str
```

Comment:
do you need this? you only use unicode when version major < 3, isn't?

Reply:
Looks we don't.

I thought `unicode` should be available when running python3. But looks like `isinstance` is not evaluated, and it doesn't require `unicode` to be in the namespace.